cmd/yurthub/app/config/config.go

```diff
@@ -181,7 +182,7 @@ func Complete(options *options.YurtHubOptions) (*YurtHubConfiguration, error) {
 		SharedFactory:             sharedFactory,
 		NodePoolInformerFactory:   dynamicSharedFactory,
 		KubeletHealthGracePeriod:  options.KubeletHealthGracePeriod,
-		FilterManager:             filterManager,
+		FilterFinder:              filterManager,
```

Comment:
Should filterManager be renamed as filterFinder?

Reply:
fixed